packages/generator/src/generator.js

```diff
@@ -31,6 +32,9 @@ export default class Generator {
       const { staticAssets } = this.options.generate
       this.staticAssetsDir = path.resolve(this.distNuxtPath, staticAssets.dir, staticAssets.version)
       this.staticAssetsBase = this.options.generate.staticAssets.versionBase
+      this.manifest = {
+        routes: []
```

Comment:
Suggestion: `precached` or so?

Reply:
Suggestion: We may also persist `staticAssets.version` here

Reply:
Actually the version is already in the staticAssetsBase

Reply:
I know it is also in page payload. Keeping version here could be beneficial to simplify on demand payload generation but nothing necessary :)